docs/integrate.md

```diff
@@ -31,40 +31,105 @@ Ultra launches the webpage inside a mobile’s web-view. Here, you may pick your
+
+### User’s login into the application
+Since the user is already logged into the Flipkart, it is not a good idea to ask user to re-login with the name and email address for entering into the external application. So, Ultra encourages automatic login to the application with user’s consent. This is achieved by using the value of `mobileNumber` parameter in combination with `isVerified` flag.
+
+!!!note
+    We suggest you to set a cookie that prevents repeated calls to the “getToken()”(link) method each time user enters into your app through Ultra.
+    Also, we highly recommend to set this towards the end of the user funnel to avoid the dropouts.
+
+This completes the login flow, user is now free to explore your app facilities.
+
+## 4: Payment
+Now comes the point where integration of your application happens with the Flipkart payment gateway (FKPG) APIs for issuing payments and refunds. For this, [contact us](contact.md) to receive the merchant credentials required to access the API. These merchant credentials vary from the parameters used for the Ultra APIs i.e. `clientId` and `secret`.
+
+The first step is to generate a [`payment token`](backend.md#payment-token) on the server side. This payment token has to be used to show the FKPG payment options screen via Ultra’s client SDK. This FKPG screen opens outside the Ultra’s container. Then, the next step is to start payment which opens the below screen for the user:
+
+![Payment](img/payments.jpg)
+
+After the payment is successful, redirect the user to your application’s order confirmation page within the container where a `POST` call leads to the `successfulCallBackUrl` specified [here](backend.md#payment-token). Check the FORM data payload details [here](backend.md#Callback-after-PGResponse). Then, issue an HTTP redirection request to the `fapp://` [redirect URL](clients#start-payment) so that the application control goes back to the Ultra container and the order confirmation page opens within this container in the similar format.
+
+The entire payment flow from initiation of payment to the order confirmation page is explained in the following sequence diagram:
+![Payments](img/image3.png)
+
+!!!note
+    It is mandatory to redirect via `fapp://` to the Ultra container.
+    You should not open any UI page of your application outside Ultra container.
+
+> Your money is transferred to an MID account which has to be configured. [Contact our team](contact.md) to onboard your MID account with us before starting the integration since it takes a significant amount of time.
+
+In case of refunds, integrate our [refund API](backend.md#refund) with your application as per your business needs.
+
+## 5: OMS Integration
+After the order has been placed and the payment has been successfully completed, the user would want to know the status of the placed order. So, now it is the time to call the [`Flipkart OMS APIs`](oms.md) to know about the order status. 
```

Comment:
Please change "...to know about the order status." to something like "...to update it with the status of the order in your application."

Reply:
Done. Please check the latest commit.